frontend/src/metabase/admin/settings/components/widgets/EmbeddingLegalese.jsx

```diff
@@ -14,7 +15,15 @@ const EmbeddingLegalese = ({ updateSetting }) =>
             If you want to hide the Metabase logo inside your own application we'd love to get in touch.
         </p> */}
         <div className="flex layout-centered mt4">
-            <button className="Button Button--primary" onClick={() => updateSetting(true)}>Enable</button>
+            <button
```

Comment:
Not a big deal either way but reminder you can do `data-metabase-event="Admin Embed Settings;Embedding Enable Click"` (though if it's on a React component rather than an HTML element we need to make sure we pass that prop down to the element itself)